docs/Plugins/Forecasting.md

```diff
@@ -0,0 +1,38 @@
+# RAT-Forecasting
+
+The forecasting plugin adds functionality to generate short-term forecasts of the reservoir state for up to 15 days. It uses forecasted weather data to forecast the inflow to reservoirs. Other Reservoir operations related fluxes, specifically, the storage change in the forecast window and the resulting outflow from the reservoir are estimated as different scenarios, which are described in detail below. 
+
+## How to Use
+To run the forecast plugin, set the value of the `forecast` option in the PLUGINS section of the configuration file to True.
+
+```
+PLUGINS: 
+	forecast: True 
+```
+
+To control the parameters related to the forecast window, the following options can be used in the BASINS section.
+
+```
+BASINS: 
```

Comment:
Should this be 'BASIN'? 
Also, won't it be better to include parameters in the Plugins section itself under forecasting subsection?

Reply:
Makes sense, updated the docs and code to reflect this change.